views/groupByAdNets.html

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+    <title>You're on the Ultra-Lightbeam</title>
+    <script src="https://d3js.org/d3.v4.min.js"></script>
+</head>
+<body>
+<p>Ads grouped by ad network average load times.</p>
```

Comment:
Rather than `ad network average load times` it should be `ad host average load times`